lib/es6-promise.umd.js

```diff
@@ -1,18 +1,16 @@
 import Promise from './es6-promise/promise';
 import polyfill from './es6-promise/polyfill';
 
-var ES6Promise = {
-  'Promise': Promise,
-  'polyfill': polyfill
-};
+Promise.Promise = Promise;
+Promise.polyfill = polyfill;
```

Comment:
Stole this from bluebird https://github.com/petkaantonov/bluebird/blob/master/src/promise.js#L771

They don't seem to do it so I wouldn't bother with it.